packages/custom-webpack/examples/full-cycle-app/karma.conf.js

```diff
@@ -1,35 +1,47 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
-module.exports = function(config) {
+const path = require('path');
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
Are you sure this is needed? `protractor` downloads it with `webdriver-manager` automatically (though we have an explicit [line of code](https://github.com/just-jeb/angular-builders/blob/master/scripts/run-ci.sh#L49) that does that in order to ensure versions alignment).

Reply:
Oh just added that because It was recommended by Karma when you not have Chrome installed on your device. I am not using Protractor.